back-end/src/App.ts

```diff
@@ -1,16 +1,20 @@
 import express from 'express';
 import https from 'https';
 import { Database } from './model/Database'
-import { router } from './routes/Router';
+import { router, httpRouter } from './routes/Router';
 import * as PassportConfig from './controllers/PassportConfig';
 import * as Conf from './Conf';
 import cors from 'cors';
 import bodyParser from 'body-parser';
 import passport from 'passport';
 
+
+
 // Create a new express application instance
 const app: express.Application = express();
 
+app.use('/http', httpRouter);
+
```

Comment:
This line has to go after `app.use(cors()); // cors is needed to allow http (axios) connection`

Otherwise you get the error:
> Access to XMLHttpRequest at 'http://localhost:3000/http/closeContactFlag' from origin 'http://localhost:8001' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Reply:
I see, apologies for not checking this earlier. Will be making the necessary changes.